cordova-lib/src/plugman/util/android-project.js

```diff
@@ -74,8 +80,8 @@ AndroidProject.prototype = {
             subProperties.dirty = true;
             this._subProjectDirs[subDir] = true;
         }
-        addToPropertyList(parentProperties, 'android.library.reference', module.exports.getRelativeLibraryPath(parentDir, subDir));
-
+        var relativePath = module.exports.getRelativeLibraryPath(parentDir, subDir);
```

Comment:
getRelativeLibraryPath can be defined as function inside the module and used without module.exports